src/tests/JIT/Regression/JitBlue/Runtime_82535/Runtime_82535.csproj

```diff
@@ -0,0 +1,11 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <OutputType>Exe</OutputType>
+    <Optimize>True</Optimize>
+    <RequiresProcessIsolation>true</RequiresProcessIsolation>
```

Comment:
Why does this require process isolation? `RequiresProcessIsolation` adds extra costs that the JIT team is actively working on reducing right now. Throwing and catching a nullref doesn't sounds like something that would need it.

Reply:
It caused a process crash on CI, which doesn't seem to happen any more.